src/frontend/Felles/Personopplysninger/PersonopplysningerMedNavKontor.tsx

```diff
@@ -21,10 +21,10 @@ const Container = styled.div`
     margin: 1rem 2rem;
 `;
 
-const PersonopplysningerMedNavKontor: React.FC<{
+export const PersonopplysningerMedNavKontor: React.FC<{
     personopplysninger: IPersonopplysninger;
     navKontor: Ressurs<INavKontor | undefined>;
-    fagsakPerson: IFagsakPerson;
+    fagsakPerson: FagsakPersonMedBehandlinger;
```

Comment:
Sjekken `harFagsak` vil nå alltid returnere true, så den må nok endres ettersom det nå blir lister som må sjekkes, og ikke strenger.

Reply:
Njaa, hvis man ikke har en fagsak på for eksempel OS vil vel feltet være `null`?
https://github.com/navikt/familie-ef-sak/blob/main/src/main/kotlin/no/nav/familie/ef/sak/fagsak/FagsakPersonController.kt#L59

Reply:
Faen, sant det - jeg som tenkte feil... Tenkte det var en liste med behandlinger, men det er jo en eller ingen fagsak, selvsagt 🤣